output/output.go

```diff
@@ -112,3 +136,33 @@ func getInputType(sourcePths []string) (string, error) {
 
 	return "", nil
 }
+
+func copyFile(source, destination string) error {
```

Comment:
This was `command.CopyFile()` from v1, but I decided to inline. I think steps should use `Exporter.ExportOutputFile()` from this file instead of manually copying a file to the deploy dir.

https://cs.github.com/?scope=org:bitrise-io&scopeName=bitrise-io&q=command.CopyFile(